testcafe/tests/helpers/utils.js

```diff
@@ -92,11 +92,22 @@ export const getLastExecutedCommand = ClientFunction(
 )
 
 //@param{string} filepath : Expected full path to file
-export async function checkLocalFile(filepath) {
-  await t.wait(data.DOWNLOAD_DELAY)
-  await t.expect(fs.existsSync(filepath)).ok(`${filepath} doesn't exist`)
-  logger.debug(`${filepath} exists on local drive`)
+export async function checkLocalFile(t, filepath) {
+  for (var i = 0; i < data.DOWNLOAD_DELAY; i++) {
+    await t.wait(10)
+    if (fs.existsSync(filepath)) {
+      logger.info(
+        `${filepath} exists on local drive. Waited for a total of ${i.toString()} microseconds`
```

Comment:
I think it's milliseconds and not microseconds, but whatever. However be aware that the `fs.existsSync` also takes time, so with the current code we will be waiting for longer than `DOWNLOAD_DELAY`. I guess `DOWNLOAD__CHECK_RETRIES` or something like that might be more accurate.

Reply:
`DOWNLOAD_DELAY` is now `DOWNLOAD_CHECK_RETRIES`